packages/zowe-explorer/src/shared/utils.ts

```diff
@@ -133,7 +135,7 @@ export function getAppName(isTheia: boolean): "Theia" | "VS Code" {
  * @param {IZoweTreeNode} node
  */
 export function getDocumentFilePath(label: string, node: IZoweTreeNode): string {
-    const dsDir = globals.DS_DIR;
+    const dsDir = globals.DS_DIR || "";
```

Comment:
I was under the impression that we set `globals.DS_DIR` during initialization in `defineGlobals` - is this logical OR still needed here?

Reply:
I think this fallback was needed for tests and I copied it from the USS implementation of `getDocumentFilePath`. Discussed this offline and ideally we would fix tests so they define globals and we could get rid of the fallback here, but it may not be worthwhile since this code is going away in v3 with the FileSystemProvider.